src/vmod_memcached.c

```diff
@@ -76,7 +76,10 @@ get_memcached(const struct vrt_ctx *ctx, struct vmod_mc_vcl_settings *settings)
 	struct timespec wait;
 
 	CHECK_OBJ_NOTNULL(settings, VMOD_MC_SETTINGS_MAGIC);
-	AN(settings->pool);
+	if (!settings->pool) {
+		VSL(SLT_Error, 0, "Could not connect to any memcached servers");
```

Comment:
In addition to logging the error, you should also fail the transaction.

Reply:
I have to admit I don't know how to do that. I'll read up.